packages/connect-explorer/src/pages/blog.mdx

```diff
@@ -0,0 +1,436 @@
+---
+auto_sections: false
+---
+
+import styled from 'styled-components';
+
+import { spacingsPx } from '@trezor/theme';
+import { Card as TrezorCard } from '@trezor/components';
+export const SectionCard = styled(TrezorCard)`
+    margin-bottom: ${spacingsPx.xl};
+`;
+
+# Create a webextension that interacts with Trezor devices using Typescriptand Webpack
+
+<SectionCard>
+    ## Introduction
+
+    In this tutorial, you will learn how to create a web extension that utilizes the @trezor/connect-webextension library.
+    By the end of this guide, you will have a functional web extension running in Chrome.
+
+</SectionCard>
+
+<SectionCard>
+    ## Prerequisites
+    Before you begin, ensure you have the following installed on your machine:
+
+    - [Node.js](https://nodejs.org/en) (we are using version 20.12.2)
+    - [Yarn](https://yarnpkg.com/) (package manager)
+    - A code editor (e.g., Visual Studio Code)
+
+</SectionCard>
+
+<SectionCard>
+    ## Set Up Your Project
+
+    1. Create a New Directory for Your Project: Open your terminal
+    and create a new directory for your web extension project.
+
+    ```bash
+    mkdir my-trezor-extension
+    cd my-trezor-extension
+    ```
+
+    2. Initialize a New Yarn Project: Run the following command to create a
```

Comment:
Yarn is not actually used

Reply:
https://github.com/trezor/trezor-suite/pull/14043/commits/695cdede30d0d770489740225a9c0c55e4f6e96e